pages/_app.tsx

```diff
@@ -10,6 +10,7 @@ import type { BasePageProps } from 'types';
 import '@prezly/uploadcare-image/build/styles.css';
 import 'modern-normalize/modern-normalize.css';
 import '../styles/styles.globals.scss';
+import '@prezly/content-renderer-react-js/styles.css';
```

Comment:
I have build error from next js - i must have global styles only in `_app.tsx`

Reply:
It's better to move this import a bit higher up, right before `uploadcare-image` styles